860_A1_Q4.ipynb

```diff
@@ -205,7 +269,7 @@
     "## create object of Linearregression class\n",
     "reg = lp()\n",
     "## Fit the linear rgerssion\n",
-    "reg = reg.fit(X_train,Y_train)\n",
+    "reg = reg.fit(X_train,Y_train) # [Paul: I think X_train has not been updated, could create subtle bug here]\n",
```

Comment:
Its a duplicate code of line..if you notice this code is repeated after I update the X_train and Y_train...I assume there should not be any bug. But thanks for pointing that out..I will update it